### DISCUSSION TITLE
Raspian 6.1.0/ bookworm issues

### DISCUSSION THREAD

**Discussion by Tobi7241** [beep-projects/SystaPi#19] [General]
with  Raspbian 1:6.1.63-1+rpt1 (2023-11-24) bookworm the SDcard boot folder is moved to /boot/firmware, so the files and scripts are not acessible. 
I setup the SystaPi manually step by step for my own education.

I have an "Raspberry Pi 3 Model B Rev 1.2" 
eth0 = local house network
eth1 = enc28j60 -> systaCofort
wlan0 = local house network (unused)

my HW is : deviceTouchInfoString  
"SC2 1 192.168.1.13 255.255.255.0 192.168.1.1 SystaComfort-II\u00000 0809780002 0 V0.28 V1.00 <mac>"

unitVersion | "1.2.2"
unitBaseVersion | "V0.28"

Is there a way to channel STouch port 3477 from eth1 into my house network, so I can use the Stouch app to see and change relevant data ?

Thanks
Tobias

**Comment by beep-projects** [comment]
Hi Tobias,

I am currently working on the script for bookworm. The boot folder is an issue and the network management has changed. 
Great that you figured this our by yourself!

For the STouch, I am trying to figure out how to properly configure nftables to support it.
Up to now I only managed to have some static routing for one device.
Make sure you read the comments at the top (I just spend half a day to figure out, that ip_forwarding got disabled on bookworm) and I have not yet looked into making these rules permanent, but I am sure google will help you 
[enable_stouch.nft.zip](https://github.com/beep-projects/SystaPi/files/13753818/enable_stouch.nft.zip)

Hope that helps,
Beep

**Reply by Tobi7241** [reply]
I'm running this script on boot to setup ethernet (eth0 = home network, eth1 =SystaSolar):

```
 sudo ip addr add 192.168.1.1/24 broadcast 192.168.1.255 dev eth1
 sleep2
 sudo ip link set eth1 down
 sleep 5
 sudo ip link set eth1 up
```

I don't have Wifi near the SystaComfort, so disable here
` sudo ip link set wlan0 down`

I setup a cron job at 11:50pm to send an daily email
```
curl -s localhost:1337/SystaREST/status > status.json

VAR_BUF= "$(jq -r '.bufferTempTop' status.json)"
VAR_GAIN="$(jq -r '.solarGainDay' status.json)"
VAR_TOT= "$(jq -r '.solarGainTotal' status.json)"
VAR_TIME="$(jq -r '.timestampString' status.json)"
```

```
sendemail -f <from_email> -t  <to_email>  -u "SystaPi report $VAR_GAIN kWh" 
 -m "Es ist $VAR_TIME, Speicher hat $VAR_BUF C und heute wurden $VAR_GAIN kWh gespeichert. Total $VAR_TOT kWh" 
 -s smtp.mail.de:587 -xu <smtp-email> -xp '<smtp-pass>' -o tls=yes
```

this works perfectly with smtp.web.de and replaces the daily summary emails from Paradigma.

Tobias
